src/app/src/components/FacilityListItemsConfirmationTableRowItem.jsx

```diff
@@ -0,0 +1,124 @@
+import { React } from 'react';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import Checkbox from '@material-ui/core/Checkbox';
+import CellElement from './CellElement';
+import ShowOnly from './ShowOnly';
+
+import {
+    listTableCellStyles,
+    confirmRejectMatchRowStyles,
+} from '../util/styles';
+
+import { makeFacilityDetailLink } from '../util/util';
+import {
+    facilityMatchStatusChoicesEnum,
+    CONFIRM_ACTION,
+} from '../util/constants';
+
+function FacilityListItemsConfirmationTableRowItem({
```

Comment:
Could you please use arrow functions whenever possible? I aim to leverage the latest features of JavaScript.

Reply:
Ok, but only for this component as this is the scope of the ticket. Other components refactoring related to the tech debt (https://opensupplyhub.atlassian.net/browse/OSDEV-838).